src/gpl/src/initialPlace.cpp

```diff
@@ -41,6 +43,40 @@
 #include "graphics.h"
 
 #include "utl/Logger.h"
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include "cusparse.h"
+#include "cusolverDn.h"
+#include "cusolverSp.h"
+
+#define CUDA_ERROR(value) do { \
+   cudaError_t m_cudaStat = value; \
```

Comment:
What does the `m` prefix signify here ?

Usually, `m_xxx` is often used as an indication for 'member variable', but looks like this is more a temporary value here.